ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/management-security.j2

```diff
@@ -129,6 +129,45 @@
 | {{ policy.name }} | > {{ min_digits }} | > {{ min_length }} | > {{ min_lower }} | > {{ min_special }} | > {{ min_upper }} | < {{ max_repetitive }} | < {{ max_sequential }} |
 {%         endfor %}
 {%     endif %}
+{%     if management_security.shared_secret_profiles is arista.avd.defined %}
+
+### Secret Profiles
+
+| Profile Name | Secret Name | Secret Password | Receive Lifetime | Transmit Lifetime |
```

Comment:
Never show the password in a table. Just remove the column.

Reply:
Removed password from table. Added new column for secret type.